src/test/java/uk/gov/hmcts/reform/pip/data/management/service/filegeneration/nonstrategic/CstWeeklyHearingListFileConverterTest.java

```diff
@@ -0,0 +1,206 @@
+package uk.gov.hmcts.reform.pip.data.management.service.filegeneration.nonstrategic;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.IOUtils;
+import org.assertj.core.api.SoftAssertions;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import uk.gov.hmcts.reform.pip.data.management.service.filegeneration.NonStrategicListFileConverter;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import java.util.Map;
+import java.util.Objects;
+
+import static uk.gov.hmcts.reform.pip.model.publication.ListType.CST_WEEKLY_HEARING_LIST;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class CstWeeklyHearingListFileConverterTest {
```

Comment:
Can we add @ActiveProfiles("test") to all the test classes in this folder?

Reply:
added